tests/h/views/api/groups_test.py

```diff
@@ -184,6 +186,78 @@ def pyramid_request(self, pyramid_request, factories):
         return pyramid_request
 
 
+@pytest.mark.usefixtures('group_service',
+                         'user_service',
+                         'request_auth_client',
+                         'validate_auth_client_authority')
+class TestAddMember(object):
+
+    def test_it_adds_current_user_to_group(self,
+                                           factories,
+                                           pyramid_request,
+                                           group_service,
+                                           user_service):
+        group = factories.Group()
+        user = factories.User()
+
+        user_service.fetch.return_value = user
+
+        pyramid_request.matchdict['userid'] = user.userid
+        pyramid_request.matchdict['pubid'] = group.pubid
+
+        resp = views.add_member(group, pyramid_request)
+
+        group_service.member_join.assert_called_once_with(group, user.userid)
+        assert isinstance(resp, HTTPNoContent)
+
+    def test_it_fails_with_mismatched_user_and_group_authorities(self,
+                                                                 factories,
+                                                                 pyramid_request,
+                                                                 group_service,
+                                                                 user_service):
+        group = factories.Group(authority="different_authority.com")
+        user = factories.User()
```

Comment:
Can you instantiate this user here with an explicit `authority` (I know it defaults to something but it's hard to tell that here) so one can see easily that the two authorities don't match?